feature/gribi/otg_tests/vrf_policy_driven_te/vrf_policy_driven_te_test.go

```diff
@@ -0,0 +1,1946 @@
+	if err := gribi.FlushAll(args.client); err != nil {
+		t.Fatal(err)
+	}
+
+	// Configure GRIBi baseline AFTs.
+	configGribiBaselineAFT(ctx, t, dut, args)
+
+	t.Run("Program gRIBi decap route", func(t *testing.T) {
+		configureGribiRoute(ctx, t, dut, args, ipv4OuterDst111+"/32")
+	})
+
+	// TODO : https://github.com/open-traffic-generator/fp-testbed-juniper/issues/42
+	// Below code will be uncommented once ixia issue is fixed.
+	/*
+		portList := []string{"port2"}
+		dstPorts := []attrs.Attributes{atePort2, atePort3, atePort4, atePort5, atePort6, atePort7, atePort8}
+
+		flow1 := createFlow(&flowArgs{flowName: "flow1", srcPort: atePort1, dstPortList: dstPorts,
+			outHdrSrcIP: ipv4OuterSrc222, outHdrDstIP: ipv4OuterDst111, outHdrDscp: []uint32{dscpEncapA1},
+			InnHdrSrcIP: atePort1.IPv4, InnHdrDstIP: ipv4InnerDst, isInnHdrV4: true, udp: true, inHdrDscp: []uint32{dscpEncapA1}})
+
+		flow2 := createFlow(&flowArgs{flowName: "flow2", srcPort: atePort1, dstPortList: dstPorts,
+			outHdrSrcIP: ipv4OuterSrc111, outHdrDstIP: ipv4OuterDst111, outHdrDscp: []uint32{dscpEncapA1},
+			InnHdrSrcIPv6: atePort1.IPv6, InnHdrDstIPv6: ipv6InnerDst, isInnHdrV4: false, udp: true, inHdrDscp: []uint32{dscpEncapA1}})
+
+		sendTraffic(t, args, portList, []gosnappi.Flow{flow1, flow2})
+		verifyTraffic(t, args, []string{"flow1", "flow2"}, !wantLoss)
+
+		captureAndValidatePackets(t, args, &packetValidation{portName: portList[0],
+			outDstIP: []string{gribiIPv4EntryVRF1111, gribiIPv4EntryVRF1112}, inHdrIP: ipv4InnerDst, validateEncap: true})
+
+		wantWeights := []float64{
+			0.0156, // 1.56 Port2
+			0.0468, // 4.68 Port3
+			0.1875, // 18.75 Port4
+			0,      // 0 Port5
+			0.75,   // 75.0 Port6
+			0,      // 0 Port7
+			0,      // 0 Port8
+		}
+		validateTrafficDistribution(t, args.ate, wantWeights)
+
+		flow3 := createFlow(&flowArgs{flowName: "flow3", srcPort: atePort1, dstPortList: dstPorts,
+			outHdrSrcIP: ipv4OuterSrc111, outHdrDstIP: ipv4OuterDst111, outHdrDscp: []uint32{dscpEncapB1},
+			InnHdrSrcIP: atePort1.IPv4, InnHdrDstIP: ipv4InnerDst, isInnHdrV4: true, udp: true, inHdrDscp: []uint32{dscpEncapB1}})
+
+		flow4 := createFlow(&flowArgs{flowName: "flow4", srcPort: atePort1, dstPortList: dstPorts,
+			outHdrSrcIP: ipv4OuterSrc222, outHdrDstIP: ipv4OuterDst111, outHdrDscp: []uint32{dscpEncapB1},
+			InnHdrSrcIP: atePort1.IPv4, InnHdrDstIP: ipv4InnerDst, isInnHdrV4: true, udp: true, inHdrDscp: []uint32{dscpEncapB1}})
+
+		sendTraffic(t, args, portList, []gosnappi.Flow{flow3, flow4})
+		verifyTraffic(t, args, []string{"flow3", "flow4"}, !wantLoss)
+
+		captureAndValidatePackets(t, args, &packetValidation{portName: args.otgConfig.Ports().Items()[1].Name(),
+			outDstIP: []string{gribiIPv4EntryVRF1111, gribiIPv4EntryVRF1112}, inHdrIP: ipv4InnerDst, validateEncap: true})
+
+		wantWeights = []float64{
+			0.0468, // 4.68 Port2
+			0.1406, // 14.06 Port3
+			0.5625, // 56.25 Port4
+			0.25,   // 25 Port5
+			0,      // 0 Port6
```

Comment:
This should be Port6 as well, not Port5 ?

Reply:
Fixed.